worker/backup_common.go

```diff
@@ -54,6 +54,8 @@ type Manifest struct {
 	// backup gets assigned the next available number. Used to verify the integrity
 	// of the data during a restore.
 	BackupNum uint64 `json:"backup_num"`
+	// Version specifies the Dgraph version, the backup was taken on.
+	Version string `json:"version"`
```

Comment:
This is the DgraphVersion. This helps in figuring out which version the backup was taken on.
I think we should use a number as we may later need >= type of check later on. We can parse it later as well.

Reply:
Use numbers instead. 2103, 2011, etc